lib/Zonemaster/Backend/Translator.pm

```diff
@@ -18,7 +18,7 @@ extends 'Zonemaster::Engine::Translator';
 sub translate_tag {
     my ( $self, $hashref ) = @_;
 
-    my $entry = Zonemaster::Engine::Logger::Entry->new( %{ $hashref } );
+    my $entry = Zonemaster::Engine::Logger::Entry->new( { %{ $hashref } } );
```

Comment:
Wouldn't this work too?

```suggestion
    my $entry = Zonemaster::Engine::Logger::Entry->new( $hashref );
```

Reply:
From what I recall, this won't work because of a deletion of keys in Zonemaster::Engine::Logger::Entry that would affect this hashref, so we need to pass a copy.

https://github.com/zonemaster/zonemaster-engine/blob/342a37aff272a7813643efe270a7c4fa94ed76d4/lib/Zonemaster/Engine/Logger/Entry.pm#L56-L58

Reply:
Oh. But in that case maybe the fix should be to change the constructor rather than the call site?